lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,141 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid lightgray;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
+
+  > div {
+    display: table-cell;
+  }
+}
+
+
+// Rules for Bookmarks Button
+.courseware-bookmarks-button {
+  width: 40%;
+  padding: 0 3px;
+  vertical-align: top;
+
+  button[aria-pressed="true"] {
```

Comment:
NIT: Instead of relying here on html attributes, we've toggled state classes such as "is-active", often making "is-inactive" the default styling state.